### PR TITLE
Dependency update - fundraising-memberships

### PR DESCRIPTION
**Pull request by Sperling-0**
- Donation/Membership form postcode maximum field length validation is updated to 16

Ticket: https://phabricator.wikimedia.org/T370617

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
 		"wmde/euro": "~1.0",
 		"wmde/clock": "~1.0",
 		"wmde/fundraising-donations": "~21.0",
-		"wmde/fundraising-memberships": "~16.1",
+		"wmde/fundraising-memberships": "dev-postcode-length",
```
Comment (moiikana) [inline composer.json:20]: this PR won't be temporary, just this branch name is temporary.
later it should be the new version (in composer.lock and maybe here as well)

Comment (Sperling-0) [reply composer.json:20]: I don't really get it. Will discuss in the daily tomorrow.

**Comment by moiikana** [comment]
TODO
you have to run `make generate-database-schema` to generate our new Database_schema.sql file and commit it too

File: .docker/database/01.Database_Schema.sql [hunk 2]
```diff
@@ -1,44 +1,8 @@
...
-ALTER TABLE payment_sofort ADD CONSTRAINT FK_15892F55BF396750 FOREIGN KEY (id) REFERENCES payment (id) ON DELETE CASCADE;
-ALTER TABLE payment_credit_card ADD CONSTRAINT FK_E75AF734BF396750 FOREIGN KEY (id) REFERENCES payment (id) ON DELETE CASCADE;
-ALTER TABLE payment_bank_transfer ADD CONSTRAINT FK_8C70B08AE5FE723C FOREIGN KEY (payment_reference_code) REFERENCES payment_reference_codes (code);
-ALTER TABLE payment_bank_transfer ADD CONSTRAINT FK_8C70B08ABF396750 FOREIGN KEY (id) REFERENCES payment (id) ON DELETE CASCADE;
-ALTER TABLE payment_paypal_identifier ADD CONSTRAINT FK_D7AFB034C3A3BB FOREIGN KEY (payment_id) REFERENCES payment (id);
-ALTER TABLE address_change ADD CONSTRAINT FK_7B0E7B9FF5B7AF75 FOREIGN KEY (address_id) REFERENCES address (id);
-ALTER TABLE bucket_log_bucket ADD CONSTRAINT FK_953634738DD5EE37 FOREIGN KEY (bucket_log_id) REFERENCES bucket_log (id);
+
+Fatal error: Uncaught Symfony\Component\Dotenv\Exception\PathException: Unable to read the "/usr/share/nginx/www/spenden.wikimedia.de/current/bin/../.env" environment file. in /usr/share/nginx/www/spenden.wikimedia.de/current/vendor/symfony/dotenv/Dotenv.php:554
```
Comment (moiikana) [inline .docker/database/01.Database_Schema.sql:2]: looks like there was an error when generating it locally

**Review by moiikana — CHANGES_REQUESTED** [review]
The database schema file needs to be fixed

**Review by gbirke — APPROVED** [review]

**Pull request merged**